lib/gnat/jetstream/api/kv.ex

```diff
@@ -265,6 +265,28 @@ defmodule Gnat.Jetstream.API.KV do
     end
   end
 
+  @doc """
+  Returns a list of all the buckets in the KV
+  """
+  @spec list_buckets(conn :: Gnat.t()) :: {:error, term()} | {:ok, list(String.t())}
+  def list_buckets(conn) do
+    case Stream.list(conn) do
+      {:ok, %{streams: streams}} ->
+        {:ok,
+         streams
```

Comment:
I would usually build a `stream_names = Enum.flat_map(...)` on it's own before assembling the `{:ok, stream_names}` tuple. That avoids extra indentation and is easier to read the flow of the function. This is a subjective  opinion for sure, but most of the codebase in this project follows this pattern, so I think it's worth applying here.